terraform/main.tf

```diff
@@ -0,0 +1,175 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.8.0"
+    }
+  }
+
+  required_version = ">= 0.14.9"
+}
+
+provider "aws" {
+  profile = "radius-tracker"
```

Comment:
for testing purposes, you can create cli credentials and put them into `radius-tracker` profile inside `~/.aws/credentials` file, or I can create credentials for you (ping me if you need them). After we replace those credentials with Rangle's.